src/featureFlags.js

```diff
@@ -6,25 +6,25 @@ import {
   UNLEASH_URL,
   UNLEASH_CLIENT_KEY,
   UNLEASH_POLLING_INTERVAL,
+  STAGE,
 } from './constants';
 
 export const Events = {
   WALLET_SERVICE_ENABLED: 'wallet-service-enabled',
 };
 
 export class FeatureFlags extends events.EventEmitter {
-  constructor(userId, network) {
+  constructor(userId) {
     super();
 
     this.userId = userId;
-    this.network = network;
-    this.walletServiceFlag = `wallet-service-mobile-${Platform.OS}-${this.network}.rollout`;
+    this.walletServiceFlag = 'wallet-service-mobile.rollout';
     this.walletServiceEnabled = null;
     this.client = new UnleashClient({
       url: UNLEASH_URL,
       clientKey: UNLEASH_CLIENT_KEY,
       refreshInterval: UNLEASH_POLLING_INTERVAL,
-      appName: `wallet-service-mobile-${Platform.OS}`,
+      appName: `wallet-service-mobile`,
```

Comment:
Change to single quote.

```suggestion
      appName: 'wallet-service-mobile',
```